app/renderer/components/preferences/payment/advancedSettings.js

```diff
@@ -56,29 +56,30 @@ class AdvancedSettingsContent extends ImmutableComponent {
           </SettingItem>
         </SettingsList>
       </div>
-      <div className={css(styles.settingsPanelDivider, styles.lastDivider)}>
-        <SettingsList className={css(commonStyles.noMarginBottom)}
-          listClassName={css(styles.list)}
-        >
+      <div className={css(styles.advancedSettings__switches)}>
+        <SettingsList className={css(commonStyles.noMarginBottom)}>
           <SettingCheckbox
             dataTestId='payment-advance-notifications'
             dataL10nId='notifications'
             prefKey={settings.PAYMENTS_NOTIFICATIONS}
             settings={this.props.settings}
             onChangeSetting={this.props.onChangeSetting}
-            className={css(styles.listItem)}
-            switchClassName={css(styles.checkboxSwitch)}
-            labelClassName={css(commonStyles.noMarginBottom)}
+            className={css(
+              styles.advancedSettings__switches__listItem,
+              styles.advancedSettings__switches__listItem_first
+            )}
+            switchClassName={css(styles.advancedSettings__switches__listItem__checkboxSwitch)}
+            labelClassName={css(styles.advancedSettings__switches__listItem__checkboxSwitch__label)}
           />
           <SettingCheckbox
             dataTestId='payment-advance-nonverified'
             dataL10nId='nonVerifiedPublishers'
             prefKey={settings.PAYMENTS_ALLOW_NON_VERIFIED}
             settings={this.props.settings}
             onChangeSetting={this.props.onChangeSetting}
-            className={css(styles.listItem, commonStyles.noMarginBottom)}
-            switchClassName={css(styles.checkboxSwitch)}
-            labelClassName={css(commonStyles.noMarginBottom)}
```

Comment:
commonStyles.noMarginBottom is no longer necessary thanks to `advancedSettings__switches__listItem_first`